src/main/java/duke/core/Parser.java

```diff
@@ -0,0 +1,128 @@
+package duke.core;
+
+import duke.command.*;
```

Comment:
I think the imported classes should be listed explicitly? (Instead of using *)

Reply:
Yes, thanks for pointing that out!